modules/creating-manifest-file-multiple-interfaces-br-ex.adoc

```diff
@@ -0,0 +1,98 @@
+// Module included in the following assemblies:
+//
+// IPI
+// * installing/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc
+// * installing/ipi-install-post-installation-configuration.adoc
+// UPI
+// * installing/installing_bare_metal/installing-bare-metal-network-customizations.adoc
+// * installing/installing_bare_metal/installing-restricted-networks-bare-metal.adoc
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+// * post_installation_configuration/post-install-bare-metal-configuration.adoc
+
+ifeval::["{context}" == "ipi-install-post-installation-configuration"]
+:postinstall-bare-metal-ipi:
+endif::[]
+ifeval::["{context}" == "post-install-bare-metal-configuration"]
+:postinstall-bare-metal-upi:
+endif::[]
+
+:_mod-docs-content-type: PROCEDURE
+[id="creating-manifest-file-multiple-interfaces-br-ex_{context}"]
+= Optional: Creating an additional OVN-Kubernetes `br-ex` bridge network
+
+You can create an additional `br-ex` bridge network that has a different configuration to the default `br-ex` bridge. Creating an additional `br-ex` bridge requires a change to the `configure-ovs.sh` shell script. Changing the shell script requires that you create a machine config manifest file for the network plugin of the Secure Neutron Networking implementation
+
+After you create and edit the manifest file, the Machine Config Operator completes tasks in the following order:
+
+* Runs the `configure-ovs.sh` shell script.
+* Drains all nodes in your cluster.
+* Inject Ignition configuration files into each node in your cluster, so that each node receives the additional `br-ex` bridge network configuration.
+* Uncordons the nodes. After all the nodes return to the `Ready` state, the Operator applies the `k8s.ovn.org/l3-gateway-config` annotation to each node.
+
+For more information about useful situations for the additional `br-ex` bridge and a situation that always requires the default `br-ex` bridge, see "Configuration for a localnet topology"
+
+.Prerequisites IF ANY?
+
+
+.Procedure
+
+. Create an Open vSwitch (OVS) bridge in the `etc/ovnk/extra_bridge` directory of your OVN-Kubernetes network plugin:
++
+.Example configuration for the `extra_bridge` file
```

Comment:
Configuration file below is not correct - the file just needs to include the iface name of the target interface:

~~~
In order to initialize a secondary bridge interface `br-ex1`, the file `/etc/ovnk/extra_bridge` must be present on the host, and include the iface name that will be supporting the bridge. Note that this file must not reference the default interface used by `br-ex` which holds the primary IP of the node:

Example configuration for the `extra_bridge` file `/etc/ovnk/extra_bridge`, below exampled referencing secondary interface attached to a vlan:

[source,text]
----
ens4f1.1001
----
~~~

Reply:
Removed the example and stated the above information. We do not need to detail how a customer uses the OVN-Kubernetes network plugin to create the `extra_bridge` file? Do commands exist?